.env-example

```diff
@@ -50,3 +50,6 @@ AWS_REGION="ap-southeast-1"
 export DD_ENV="local"
 export DD_SERVICE="isomer"
 export DD_TAGS="service:isomer"
+
+#Cloudmersive
```

Comment:
super ultra nit: can add an extra space? 

also, do take note that we need to update the 1PW

Reply:
Hey good catch, I did the right spacing in my local .env but forgot to enforce it in .env-example. Also fixed all comment space after double slash issue. 
Fixed it in 6ebc12a9cfe977f4a309329fc571ae04322bb93c